Week-3/CSS-ADVANCED/q1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Question and Answer</title>
+    <style>
+
+        body {
+         font-family: Calibri, sans-serif;
+         font-size: 3em;
+      }
+
+      #answer {
+         color: white;
+         width: 100px;
+         animation-name: moveFraction;
+         animation-duration: 2s;
+         animation-delay: 1s;
+         animation-fill-mode: forwards;
+
+      }
+
+      @keyframes moveFraction {
+         0% {
+           color: yellow;
+           transform: translate(-355px, 60px);
+         }
+         50% {
+           transform: translate(0px, 60px);
+         }
+         100% {
+           color: red;
+           transform: translate(0px, 0px);
+         }
+       }
+
+
+
+      #question {
+         transition: transform 0.6s ease-in-out;
+
+      }
+
+      #question:hover {
+         transform: scale(0.9);
+
+      }
+
+      span {
```

Comment:
It's a good practice to use classes instead of element selectors for styling, especially for elements like `span `that are very common.